pkg/syncer/spec/spec_process.go

```diff
@@ -114,14 +114,37 @@ func (c *Controller) process(ctx context.Context, gvr schema.GroupVersionResourc
 	}
 	clusterName, name := clusters.SplitClusterAwareKey(clusterAwareName)
 
-	// to downstream
-	downstreamNamespace, err := shared.PhysicalClusterNamespaceName(shared.NamespaceLocator{
-		LogicalCluster: clusterName,
-		Namespace:      upstreamNamespace,
-	})
+	namespaceGvr := schema.GroupVersionResource{Group: "", Version: "v1", Resource: "namespaces"}
```

Comment:
(nit, style) always define variables/constants as near as possible to their use.

Reply:
done